en/services/parachute.md

```diff
@@ -0,0 +1,158 @@
+| <span id="PARACHUTE_SAFETY_FLAGS"></span>[PARACHUTE_SAFETY_FLAGS](../messages/common.md#PARACHUTE_SAFETY_FLAGS) | Parachute module safety-related flags. |
+| <span id="PARACHUTE_ERROR_FLAGS"></span>[PARACHUTE_ERROR_FLAGS](../messages/common.md#PARACHUTE_ERROR_FLAGS) | Fault/health indications. |
+
+## Implementation and Messages
+
+### Parachute Connection
+
+Parachutes are expected to follow the [Heartbeat/Connection Protocol](https://github.com/mavlink/mavlink-devguide/blob/master/en/services/heartbeat.md) and send a constant flow of heartbeats (nominally at 1Hz). Parachutes are identified via their type [MAV_TYPE_PARACHUTE](#MAV_TYPE_PARACHUTE).
+Individual parachutes are distinguished via their unique component ID, which by default should be [MAV_COMP_ID_PARACHUTE](#MAV_COMP_ID_PARACHUTE) (though this is not mandated and any ID may be used).
+
+Once a heartbeat is received, the drone can then send a [MAV_CMD_REQUEST_MESSAGE](https://mavlink.io/en/messages/common.html#MAV_CMD_REQUEST_MESSAGE) command to the parachute to receive information about its status. [MAV_CMD_SET_PARACHUTE_ARM](#MAV_CMD_SET_PARACHUTE_ARM) can also then be sent to configure the arming status of the various possible trigger mechanisms of the parachute. 
+
+
+### STATUS
+The [PARACHUTE_STATUS](#PARACHUTE_STATUS) message can be requested to receive information about the status of the parachute. This includes information such as uptime, errors, arm status, deployment status, safety status, arm altitiude, and parachute pack date.
+
+The parameter list can be found below:
+| Parameter                | Description                                                                                                                                    |
+|--------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| `time_boot_ms`           | Time since system boot up in milliseconds.                                                                                                     |
+| `error_status`           | Bitmask detailing the [PARACHUTE_ERROR_FLAGS](#PARACHUTE_ERROR_FLAGS).                                                                         |
+| `arm_status`             | Bitmask detailing the arming status via [PARACHUTE_TRIGGER_FLAGS](#PARACHUTE_TRIGGER_FLAGS).                                                   |
+| `deployment_status`      | Parachute deployment status.                                                                                                                   |
+| `safety_status`          | Parachute safety status.                                                                                                                       |
+| `ats_arm_altitude`       | Parachute Automatic Trigger System (ATS) auto-arming/disarming altitude in meters.                                                             |
```

Comment:
what is this? Is this the MDA? I don't think the calling it the "auto-arming/disarming altitude" is accurate?

Reply:
This is not used in our product but is something in the protocol we are upstreaming.